code/game/objects/items/RCD.dm

```diff
@@ -1082,7 +1082,7 @@ GLOBAL_VAR_INIT(icon_holographic_window, init_holographic_window())
 ///The plumbing RCD. All the blueprints are located in _globalvars > lists > construction.dm
 /obj/item/construction/plumbing
 	name = "Plumbing Constructor"
-	desc = "An expertly modified RCD outfitted to construct plumbing machinery."
+	desc = "An expertly modified RCD outfitted to construct plumbing machinery. Alt-Click to change layer and duct color."
```

Comment:
I think it'd be better to put this on the examine proc under a span_notice(), but i'll leave it up to you.